build.gradle

```diff
@@ -58,6 +58,7 @@ buildscript {
       firebaseCore        : '16.0.8',
       firebaseConfig      : '16.4.1',
       googleServices      : '4.0.1',
+      playCore            : '1.5.0'
```

Comment:
The current latest version on [Google Maven](https://maven.google.com) is **1.6.1**. The developer docs don't usually point to the latest version. Use this version instead?